compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -108,7 +141,8 @@ class SkikoComposeUiTest(
 
     private val composeRootRegistry = ComposeRootRegistry()
 
-    private val coroutineDispatcher = UnconfinedTestDispatcher()
+    @InternalTestApi
+    val coroutineDispatcher = UnconfinedTestDispatcher()
```

Comment:
I would keep it `private`. Otherwise, we can come up with more complicated data flow in the future when we start to misuse it.

Alternative - you can pass it via constructor, or just create a separate dispatcher in the test.